aw_server/rest.py

```diff
@@ -278,9 +278,18 @@ class ImportAllResource(Resource):
     @api.expect(buckets_export)
     @copy_doc(ServerAPI.import_all)
     def post(self):
-        data = request.get_json()
-        buckets = data["buckets"]
-        return app.api.import_all(buckets), 200
+        # If import comes from a form in th web-ui
+        if len(request.files) > 0:
```

Comment:
Not really sure how I'm supposed to add support for this in aw-server-rust, I will have to investigate that.